src/java.base/share/classes/java/io/FileFilter.java

```diff
@@ -29,9 +29,9 @@
 /**
  * A filter for abstract pathnames.
  *
- * <p> Instances of this interface may be passed to the <code>{@link
- * File#listFiles(java.io.FileFilter) listFiles(FileFilter)}</code> method
- * of the <code>{@link java.io.File}</code> class.
+ * <p> Instances of this interface may be passed to the {@link
+ * File#listFiles(java.io.FileFilter) listFiles(FileFilter)} method
```

Comment:
I would tend to keep {@link on the same line with the rest of the link.

Reply:
Will change before integrating.